src/components/Header/TextForm.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+class TextForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ""
+    };
+  }
+
+  handleChange = event => {
+    this.setState({ value: event.target.value });
```

Comment:
Don't repeat `event.target.value`. Use destructuring and create a constant for `value` and use that here and below